_observing-your-data/trace/ta-dashboards.md

```diff
@@ -1,25 +1,143 @@
+
+  ![Services throughput tooltip ]({{site.url}}{{site.baseurl}}/images/trace-analytics/service-throughput-tooltip.png)
+  
+  - The **Services** correlation dialog window provides an at-a-glance overview of a service's details, including its 24-hour throughput trend. You can use these details to analyze correlated logs or traces by filtering based on the `serviceName` field. The following image shows this window.
+ 
+  ![Services correlation dialog window]({{site.url}}{{site.baseurl}}/images/trace-analytics/single-service-correlation-dialog.png)
+
+  - The **Services** RED metrics dialog window provides an at-a-glance overview of a service's RED metrics indicators, including 24-hour error, duration, and throughput rate. The following image shows this window.
+ 
+  ![Services RED metrics for duration]({{site.url}}{{site.baseurl}}/images/trace-analytics/single-service-RED-metrics.png)
+
+  - The **Span details** dialog window provides a trace's details. You can use this information to further analyze a trace's elements, such as attributes and associated logs. The following image shows this window.
+
+  ![Services Span details dialog window]({{site.url}}{{site.baseurl}}/images/trace-analytics/span-details-fly-out.png)
+
+- **Service map**
+  - The **Service map** displays nodes, each representing a service. Node color indicates the RED indicator severity for that service and its dependencies. The following image shows a map.
+
+  ![Services map tooltip]({{site.url}}{{site.baseurl}}/images/trace-analytics/service-details-tooltip.png)
+  
+  - Selecting a node opens a detailed dialog window for that service. This interactive map visualizes service interconnections, helping identify problems by service, unlike dashboards that identify issues by operation. Sort by error rate or latency to pinpoint potential problem areas.
+
+  - In the **Service map** dialog window, nodes represent connected downstream services dependent on the selected service. Node color indicates the RED indicator severity for that service and its downstream dependencies. The following image shows this dialog window.
+  
+  ![Service map dialog window]({{site.url}}{{site.baseurl}}/images/trace-analytics/single-service-fly-out.png)
+
+- **Trace groups**
+  - Traces are grouped by their HTTP API name, allowing clustering based on their business functional unit. Traces are grouped by HTTP method and path, displaying the average latency, error rate, and trends associated with a particular operation. You can filter by trace group name. The following image shows the **Trace Groups** window.
+
+  ![Trace Groups window]({{site.url}}{{site.baseurl}}/images/trace-analytics/trace-group-RED-metrics.png)
+
+  - In the **Trace Groups** window, you can filter by group name and other filters. You can also analyze associated traces. To drill down on the traces that make up a group, select the number of traces in the right column, then choose an individual trace for a detailed summary.
```

Comment:
At the end of the first sentence, "filters" should be the noun that also describes "group name". Same comment re: "drill down on": "review/examine/inspect"?